src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -418,6 +430,25 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
                                custom_locations_oid, helm_client_location, enable_private_link, arm_metadata, onboarding_timeout, container_log_path)
     return put_cc_response
 
+def validate_existing_provisioned_cluster_for_reput(cluster_resource, kubernetes_distro, kubernetes_infra, enable_private_link, private_link_scope_resource_id, distribution_version, azure_hybrid_benefit, location):
+    if cluster_resource.kind == "ProvisionedCluster":
+        if azure_hybrid_benefit is not None:
+            raise InvalidArgumentValueError("Updating the 'azure hybrid benefit' property of a Provisioned Cluster is not supported from the Connected Cluster CLI. Please use the 'az aksarc update' CLI command.\nhttps://learn.microsoft.com/en-us/cli/azure/aksarc?view=azure-cli-latest#az-aksarc-update")
```

Comment:
>azure hybrid benefit [](http://example.com/codeflow?start=59&length=20)

Any reason for explicit validation and error message for this property only?  Especially since you are validating this as part of the validation_values array as well?

Reply:
see comment above